### DISCUSSION TITLE
Change log level for backward queue input

### DISCUSSION THREAD

**Discussion by my-umd** [blakeblackshear/frigate#6491] [General]
Currently, backward queue input message is logged as 'ERROR'.  E.g.,
`2023-05-14 23:48:59.367366687  [2023-05-14 19:48:59] ffmpeg.driver_way.detect       ERROR   : [aac @ 0x55f4c3057640] Queue input is backward in time

2023-05-14 23:48:59.369097438  [2023-05-14 19:48:59] ffmpeg.driver_way.detect       ERROR   : [segment @ 0x55f4c3052800] Non-monotonous DTS in output stream 0:1; previous: 538105, current: 537937; changing to 538106. This may result in incorrect timestamps in the output file.`

Such logs can easily flood the log file, making it really hard to spot important issues.  I do not know how important it is to save them.  Is it possible to change such log level to 'DEBUG' so that they are not regularly kept in the log?

**Comment by NickM-27** [comment]
All ffmpeg logs are printed under error as that is the only case that they will be printed. There is no way to control which ffmpeg logs are printed to which log level. If you want to remove them then you should adjust the global args for ffmpeg

**Reply by my-umd** [reply]
I see.  Thanks.
